src/main/java/de/zalando/aruha/nakadi/controller/EventStreamController.java

```diff
@@ -163,6 +163,9 @@ public StreamingResponseBody streamEvents(
                 if (consumerCounter != null) {
                     consumerCounter.dec();
                 }
+                if (eventConsumer != null) {
```

Comment:
Hey, I always do that! :)
You probably mean that I put `else` or `catch` on the next line. Like
```
if () {
    ....
}
else {
    ....
}
```